devdocs/settingUp.md

```diff
@@ -64,7 +64,7 @@ Important: When a version is specified, please install that version instead of t
    This value must be a valid **JDK 1.7** directory.<br>
    * `src/main/resources/build.properties`<br>
    For now, property values can remain as they are.
-   If you want to use SendGrid for developing and testing email features, create a free SendGrid account, create an API key, and update the relevant values in `build.properties`
+   If you want to use alternative email services to develop and test email features, refer to [this document](emails.md).
```

Comment:
Clearly mention this is optional. We don't want to unduly worry folks setting up for the first time.  
`[Optional] If you want ...`
In fact, we can even consider separating out all the optional/conditional steps so that the description of the initial setting up remains short.